client/devdocs/design/playground-scope.js

```diff
@@ -117,3 +117,4 @@ export VerticalMenu from 'components/vertical-menu';
 export VerticalNav from 'components/vertical-nav';
 export VerticalNavItem from 'components/vertical-nav/item';
 export Wizard from 'components/wizard';
+export WizardProgressBar from 'components/wizard-progress-bar';
```

Comment:
Do we still need this export since we decided not to include this new component in the playground?